pkg/csi/service/mounter/mounter_windows.go

```diff
@@ -0,0 +1,475 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mounter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/davecgh/go-spew/spew"
+	disk "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1"
+	diskclient "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
+
+	fs "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1"
+	fsclient "github.com/kubernetes-csi/csi-proxy/client/groups/filesystem/v1"
+
+	volume "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1"
+	volumeclient "github.com/kubernetes-csi/csi-proxy/client/groups/volume/v1"
+
+	"k8s.io/klog/v2"
+	"k8s.io/mount-utils"
+	utilexec "k8s.io/utils/exec"
+)
+
+// black assignment is used to check if it can be cast
+var _ CSIProxyMounter = &csiProxyMounter{}
+
+type csiProxyMounter struct {
+	Ctx          context.Context
+	FsClient     *fsclient.Client
+	DiskClient   *diskclient.Client
+	VolumeClient *volumeclient.Client
+}
+
+// CSIProxyMounter extends the mount.Interface interface with CSI Proxy methods.
+// In future this functions are supposed to be implemented in mount.Interface for windows
+type CSIProxyMounter interface {
+	mount.Interface
+	ExistsPath(path string) (bool, error)
+	FormatAndMount(source, target, fstype string, options []string) error
+	Rmdir(path string) error
+	MakeDir(pathname string) error
+	Rescan() error
+	GetDeviceNameFromMount(mountPath string) (string, error)
+	GetVolumeSizeInBytes(devicePath string) (int64, error)
+	ResizeVolume(devicePath string, sizeInBytes int64) error
+	GetAPIVersions() string
+	GetDiskNumber(diskID string) (string, error)
+	Mount(source string, target string, fstype string, options []string) error
+	Unmount(target string) error
+	IsLikelyNotMountPoint(path string) (bool, error)
+	GetDiskTotalBytes(devicePath string) (int64, error)
+	StatFS(path string) (available, capacity, used, inodesFree, inodes, inodesUsed int64, err error)
+}
+
+func NewSafeMounter(ctx context.Context) (*mount.SafeFormatAndMount, error) {
+	csiProxyMounter, err := newCSIProxyMounter(ctx)
+	if err == nil {
+		klog.V(2).Infof("using CSIProxyMounterV1, %s", csiProxyMounter.GetAPIVersions())
```

Comment:
Can you make sure not to use klog in this repository?
We need to use the uber zap logging framework.

Reply:
done